### ISSUE TITLE
ES6 Rollup with ThunderJS import issue 

### ISSUE THREAD

**Issue by adrianM27** [WebPlatformForEmbedded/ThunderJS#5]
In a project created by Lightning-CLI, I've tried to import and use ThunderJS:
in App.js:
`import thunderJS from 'ThunderJS'`
and just use it globally:
`const config = {port: 9998, debug: true};
const thunderJs = thunderJS(config);`
But whenever I build my app: `lng build` and run http server, the app fails with error: Uncaught ReferenceError: events$2 is not defined

Seems that rollup does strange things, since AppBundle.js starts with:
`var APP_xxx = (function (events$2, https, http, net, tls, crypto, url, zlib, fs, path, os, stream$1) {`

**Comment by wouterlucas** [comment]
I dont think this is a thunderjs issue, but likely a rollup config issue on the SDK / CLI side. 
cc @michielvandergeest
